editoast/src/models/stdcm_search_environment.rs

```diff
@@ -121,6 +118,10 @@ pub struct StdcmSearchEnvironment {
     pub default_speed_limit_tag: Option<String>,
     #[model(remote = "Vec<Option<String>>")]
     pub operational_points_id_filtered: OperationalPointIds,
+    /// Map of a gauge with their authorized track section ids.
```

Comment:
Are we really "loading gauge" dependent here? Because if that's the case, maybe the feature is not so generic afterall?